moveit_ros/planning_interface/move_group_interface/src/move_group.cpp

```diff
@@ -672,18 +677,38 @@ class MoveGroup::MoveGroupImpl
 
   MoveItErrorCode execute(const Plan &plan, bool wait)
   {
-    moveit_msgs::ExecuteKnownTrajectory::Request req;
-    moveit_msgs::ExecuteKnownTrajectory::Response res;
-    req.trajectory = plan.trajectory_;
-    req.wait_for_execution = wait;
-    if (execute_service_.call(req, res))
```

Comment:
Since we are no longer using ``execute_service_`` within move_group, I think we can remove it from this class (but keep the capability in general for backwards compatibility)

e.g. we don't need ``execute_service_`` as a MoveGroup member variable and we don't need to initialize it